### PR TITLE
Feature/welcome board

### PR DESCRIPTION
**Pull request by acarrillo3**
# Description

As a user, I want to go to the dashboard after logging in where I can reserve a room

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Change Status

- [ ] Complete, tested, ready to review and merge
- [x] Complete, but not tested (may need new tests)
- [ ] Incomplete/work-in-progress, PR is for discussion/feedback

# Has This Been Tested?

- [ ] Yes
- - [ ] What was tested?
- [x] No

# Checklist

- [ ] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] My code has been reviewed and commented by at least two peers
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [x] There are no merge conflicts

### CODE REVIEW

**Review by DrakeAlia — APPROVED** [review]

File: src/components/pages/WelcomeBoard/WelcomeBoard.css [hunk 1]
```diff
@@ -0,0 +1,33 @@
+.dashboard {
```
Comment (antonyk) [inline src/components/pages/WelcomeBoard/WelcomeBoard.css:1]: Looks good... but maybe consider writing the CSS as Less?

File: src/components/pages/WelcomeBoard/index.js [hunk 2]
```diff
@@ -0,0 +1,21 @@
+import React from 'react';
```
Comment (antonyk) [inline src/components/pages/WelcomeBoard/index.js:1]: Looks good, however, what is the thinking behind using a class component? The React team recommends using hooks on new projects.

**Review by antonyk — CHANGES_REQUESTED** [review]
Good job! Please see my comments re Less and class component.

File: src/components/pages/WelcomeBoard/index.js (hunk 2 above)
Comment (acarrillo3) [reply src/components/pages/WelcomeBoard/index.js:1]: Very true! Thanks for catching that

**Review by antonyk — APPROVED** [review]
looks good

**Pull request merged**